src/application-constants.ts

```diff
@@ -79,6 +79,11 @@ export const BLACKDUCK_FIXPR_USE_UPGRADE_GUIDANCE_KEY = 'blackduck_fixpr_useUpgr
  */
 export const BLACKDUCK_AUTOMATION_PRCOMMENT_KEY = 'blackduck_automation_prcomment'
 export const BLACKDUCK_PRCOMMENT_ENABLED_KEY = 'blackduck_prComment_enabled'
+export const BLACKDUCK_REPORTS_SARIF_CREATE_KEY = 'blackduck_reports_sarif_create'
+export const BLACKDUCK_REPORTS_SARIF_FILE_PATH_KEY = 'blackduck_reports_sarif_file_path'
+export const BLACKDUCK_REPORTS_SARIF_ISSUE_TYPES_KEY = 'blackduck_reports_sarif_issue_types'
```

Comment:
Do we actually need this? I believe issue_types is only applicable for polaris.

Reply:
removed the constant, it was not being used anywhere